src/bot/commands/approve.ts

```diff
@@ -579,13 +669,13 @@ export default class ApproveCommand extends Command {
       proposal.suggestion &&
       proposal.suggestion.approvals.length >= neededSuggestionsApprovals
     ) {
-      await this.approveSuggestion(
+      await this.approveSuggestion({
         interaction,
-        proposal.suggestion as Suggestion,
-        suggestionMessage,
-        suggestionMessage.embeds[0].toJSON(),
-        true
-      );
+        proposal: proposal.suggestion as Suggestion,
```

Comment:
Pas nécessaire vu que t'as un typage paramétré pour les options du approveSuggestion
```suggestion
        proposal: proposal.suggestion,
```